src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -753,7 +753,7 @@ void EnIk_PlayCutscene(EnIk* this, GlobalContext* globalCtx) {
             Play_CameraSetAtEye(globalCtx, this->subCamId, &this->actor.focus.pos, &subCamEye);
         }
         if (this->actor.colChkInfo.health != 0) {
-            EnIk_SetupReactToAttack(this, 0);
+            EnIk_SetupReactToAttack(this, false);
```

Comment:
(commenting here since I can't reach the actual function in the diff)

Need to change arg1 to act boolean in the function too (i.e. the != 0). Be good if you could work out what it actually does, too.